app/views/worldwide_organisations/_worldwide_organisation.html.erb

```diff
@@ -1,29 +0,0 @@
-<%= content_tag_for(:div, worldwide_organisation) do %>
-  <% worldwide_organisation.sponsoring_organisations.each do |organisation| %>
```

Comment:
Thanks for tidying up! Would it be possible to put reference to the commit that removed the last ref to this partial in the commit message? I was just having a trawl through to see what had happened here but wondered if you'd already done this digging